docs/constitution.rst

```diff
@@ -0,0 +1,118 @@
+Mu's Constitution
+=================
+
+Our Values
+----------
+
+Happily, Mu's users and developers are a diverse and international community
+with a reputation as a friendly, welcoming and dynamic group.
+
+We trust that such collaborators will treat each other in a way that reflects
+the widely held view that diversity and friendliness are strengths of our
+community to be celebrated and fostered.
+
+We believe everyone involved in Mu's community has a right to:
+
+* be treated with courtesy, dignity and respect;
+* be free from any form of discrimination, victimisation, harassment or
+  bullying;
+* enjoy an environment free from unwelcome behaviour, inappropriate language
+  and unsuitable imagery.
+
+To this end, we ask and hope participants in our community will abide by the
+spirit of our :doc:`code_of_conduct`.
+
+The Role of Administrators
+--------------------------
+
+Administrators are volunteers who serve the Mu project. They demonstrate and
+are expected to be the embodiment of our values described above.
+
+Because of such service and trust in their behaviour, they have the privileges
+needed to run the infrastructure relating to Mu.
+
+Administrators have privileges in three areas:
+
+1. The development of Mu (by virtue of their sustained and significant
+   technical contributions),
+2. The infrastructure that supports the development of Mu (technical services
+   hosted by third parties),
+3. The official channels for community discussion and participation 
+   (communication services hosted by third parties).
+
+Anything not covered by these three areas is outside the ability of the
+administrators to effectively monitor and supervise.
+
+* Admin processes for: dev, infrastructure, communication channels.
+* Who are currently administrators
+* Becoming an admin
+* Removal of an admin
+
+Contributing to Mu
+------------------
+
+We're delighted for people to report problems, make suggestions and contribute
+code. Indeed we encourage it. We'd ask you to follow our development guidelines
+and, of course, to be respectful of our time.
+
+It may be that we decide not to adopt some change you've proposed -- or even
+coded. Please don't take this personally. Any project, especially one with many
+users, has to measure contributions against its design ideals. And sometimes
+the two don't match.
+
+Managing Expectations
+---------------------
+
+We recognise that Mu is a useful educational tool for individual learners and
+also in classrooms and coding club situations. And we're delighted that the
+tool we've built helps both beginner programmers and educators as much as it
+does.
+
+However, Mu is no-one's full-time job. No-one earns money from developing Mu
+and all of us involved offer our time freely as a benefit to the educational
+coding community and those taking thier first steps in programming. The amount
```

Comment:
There's a tiny typo here. Adding a second 'to' makes it easier to parse for (non-native speaker) me.

```suggestion
coding community and to those taking their first steps in programming. The amount
```